tests/test_scan_image_tiff.py

```diff
@@ -5,7 +5,12 @@
 
 from hdmf.testing import TestCase
 from numpy.testing import assert_array_equal
-from ScanImageTiffReader import ScanImageTiffReader
+import platform
+
+if (
+    platform.processor() != "arm"
```

Comment:
Are you sure it's ALL arm that they don't support? I thought it was just mac + arm

Reply:
I don't have a windows-arm machine handy, so I have no idea if it's mac + arm or just arm.  But for the sake of CI, does it really matter?

Reply:
Well, if you wanted to specify JUST "CI" then you'd also fetch the CI environment variable from GitHub actions. The logic you have here affects every platform and device that might want to run the test suite, whereas we want a pretty narrow exclusion here